### DISCUSSION TITLE
Question about the bounty on Chrome/Firefox extension store

### DISCUSSION THREAD

**Discussion by signed-log** [DandelionSprout/adfilt#648] [Q&A] [answered]
Hello,

I've been through the bounty [list](https://github.com/DandelionSprout/adfilt/search?q=bounty&type=code), and have found some issues that can't be resolved directly on browser

Chrome Webstore on Chromium-based browsers : Not a single extension can operate on the webstore, this is known as [Privileged URLs][1], and this is fully hardcoded, so minus a rebuild or a fork of Chromium that has those disabled, there is no luck for any `chrome.google.com/webstore*` filter to work

Mozilla Addons store on Firefox : Firefox also has the concept of hardcoded privileged URL but they can be overriden [without the browser being rebuilt][1], but this can be dangerous

[1]: https://github.com/gorhill/uBlock/wiki/Privileged-Pages

**Comment by DandelionSprout** [comment, marked as answer]
The part about Firefox was certainly news to me, even if I felt the uBlock Origin wiki guide would be better if it mentioned that only `addons.mozilla.com` would need to be removed from `extensions.webextensions.restrictedDomains` (which'd hopefully prevent rogue extensions from accessing account info, compared to emptying the string) for typical users of Anti-Malware List, or of any other lists out there for that matter. I've now added procedures for it to the main version of Anti-Malware List.

I accept defeat for Chrome Web Store.
